src/Infrastructure.EntityFramework/Repositories/OrganizationDomainRepository.cs

```diff
@@ -74,32 +74,25 @@ public async Task<OrganizationDomainSsoDetailsData> GetOrganizationDomainSsoDeta
 
         using var scope = ServiceScopeFactory.CreateScope();
         var dbContext = GetDatabaseContext(scope);
-        var ssoDetails = await dbContext.Organizations
-            .Join(dbContext.OrganizationDomains, o => o.Id, od => od.OrganizationId,
-                (organization, domain) => new { resOrganization = organization, resDomain = domain })
-            .Join(dbContext.Policies, o => o.resOrganization.Id, p => p.OrganizationId,
-                (combinedOrgDomain, policy)
-                    => new
-                    {
-                        Organization = combinedOrgDomain.resOrganization,
-                        Domain = combinedOrgDomain.resDomain,
-                        Policy = policy
-                    })
-            .Select(x => new OrganizationDomainSsoDetailsData
-            {
-                OrganizationId = x.Organization.Id,
-                OrganizationName = x.Organization.Name,
-                SsoAvailable = x.Organization.UseSso,
-                OrganizationIdentifier = x.Organization.Identifier,
-                SsoRequired = x.Policy.Enabled,
-                VerifiedDate = x.Domain.VerifiedDate,
-                PolicyType = x.Policy.Type,
-                DomainName = x.Domain.DomainName,
-                OrganizationEnabled = x.Organization.Enabled
-            })
-            .Where(y => y.DomainName == domainName
-                        && y.OrganizationEnabled == true
-                        && y.PolicyType.Equals(PolicyType.RequireSso))
+        var ssoDetails = await (from o in dbContext.Organizations
+                                from od in o.Domains
+                                join s in dbContext.SsoConfigs on o.Id equals s.OrganizationId into sJoin
+                                from s in sJoin.DefaultIfEmpty()
+                                join p in dbContext.Policies.Where(p => p.Type == PolicyType.RequireSso) on o.Id
```

Comment:
❓  Forgive my ignorance here, but do we need to update this to be a left join like we have on the `OrganizationDomainSsoDetails_ReadByEmail` stored proc?